grid2op/Backend/backend.py

```diff
@@ -240,6 +242,16 @@ def cannot_handle_more_than_2_busbar(self):
                           "'fix' this issue, you need to change the implementation of your backend or "
                           "upgrade it to a newer version.")
         self.n_busbar_per_sub = DEFAULT_N_BUSBAR_PER_SUB
+
+    def set_shedding(self, allow_shedding:bool=False):
+        """
+        Override if the Backend supports shedding.
+        """
+        
+        if allow_shedding:
+            raise BackendError("Backend does not support shedding")
+        else:
+            self.allow_shedding = allow_shedding
```

Comment:
Can you make this argument "private" by convention in python name should start with `_` for "private" attribute.
So this would be `self._allow_shedding = ...`